common/reviews/api/heft.api.md

```diff
@@ -225,6 +231,7 @@ export interface IHeftTaskSession {
     readonly hooks: IHeftTaskHooks;
     readonly logger: IScopedLogger;
     readonly parameters: IHeftParameters;
+    readonly parsedCommandLine: IHeftParsedCommandLine | undefined;
```

Comment:
Hmm, is there any way we can make it so that this isn't undefined? That feels like an odd API contract to provide the dev.

Reply:
Good catch. I had already made it non-`undefined`, but just forgot to update the `IHeftTaskSession` definition.  Fixed